build-scripts/ubuntu-2004/build-3rd-parties.sh

```diff
@@ -27,6 +27,10 @@ function build_from_pypi {
 
     if [ -z $2 ]; then
         PACKAGE_VERSION=""
+        # Get the most recent package version from PyPI to be included in the package name of the Debian artifact
+        curl -X GET "https://pypi.org/pypi/${PACKAGE_NAME}/json" > "${PACKAGE_NAME}.json"
+        PACKAGE_VERSION="==$(cat "${PACKAGE_NAME}.json" | jq '.info.version' | sed -e 's/^"//' -e 's/"$//')"
```

Comment:
```suggestion
        PACKAGE_VERSION="==$(cat "${PACKAGE_NAME}.json" | jq --raw-output '.info.version')"
```
The `--raw-output` flag will provide the value for `.info.version` without the `"` eliminating the need for the `sed` filtering.

Reply:
thanks for that hint